Practice_Shop/PracticeShop.BLL/Services/Interfaces/ICategoryService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using PracticeShop.BLL.DTOs.Category;
+using PracticeShop.DAL.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PracticeShop.BLL.Services.Interfaces
+{
+    public interface ICategoryService
+    {
+        Task<IEnumerable<Category>> GetAllCategories();
```

Comment:
you need to retutn IEnumerable<CategoryResponseDto> for example. You need to create dto